engine/pom.xml

```diff
@@ -807,6 +807,7 @@
                 <exclude>**/removaltime/cleanup/HistoryCleanupScheduler*Test.java</exclude>
                 <exclude>**/HistoryCleanupDisabledOnBootstrapTest.java</exclude>
                 <exclude>**/LoginAttemptsTest.java</exclude>
+                <exclude>**/ConcurrentHistoryCleanupUpdateOfFailingJobTest.java</exclude>
```

Comment:
We can (optionally) add a comment here why this test is ignored for this profile.